tools/Microsoft.Health.Dicom.SchemaManager/ApplyCommand.cs

```diff
@@ -0,0 +1,62 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.CommandLine;
+using System.CommandLine.NamingConventionBinder;
+using EnsureThat;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Dicom.SchemaManager.Properties;
+using Microsoft.Health.SqlServer.Features.Schema.Manager;
+using Microsoft.Health.SqlServer.Features.Schema.Manager.Model;
+
+namespace Microsoft.Health.Dicom.SchemaManager;
+
+public class ApplyCommand : Command
+{
+    private readonly ISchemaManager _schemaManager;
+    private readonly ILogger<ApplyCommand> _logger;
+
+    public ApplyCommand(
+        ISchemaManager schemaManager,
+        ILogger<ApplyCommand> logger)
+        : base("apply", Resources.ApplyCommandDescription)
+    {
+        AddOption(CommandOptions.ConnectionStringOption());
+        AddOption(CommandOptions.ManagedIdentityClientIdOption());
+        AddOption(CommandOptions.AuthenticationTypeOption());
+        AddOption(CommandOptions.VersionOption());
+        AddOption(CommandOptions.NextOption());
+        AddOption(CommandOptions.LatestOption());
+        AddOption(CommandOptions.ForceOption());
+
+        AddValidator(commandResult => MutuallyExclusiveOptionValidator.Validate(commandResult, new List<Option> { CommandOptions.VersionOption(), CommandOptions.NextOption(), CommandOptions.LatestOption() }, Resources.MutuallyExclusiveValidation));
+
+        Handler = CommandHandler.Create(
+            (MutuallyExclusiveType type, bool force, CancellationToken token)
+            => ApplyHandler(type, force, token));
+
+        EnsureArg.IsNotNull(logger, nameof(logger));
```

Comment:
nit: just do this first to fail fast.

Reply:
Good point, thanks! Fixed in latest commit.